notifications/manager.go

```diff
@@ -149,14 +149,17 @@ func (m *Manager) Run(ctx context.Context) error {
 		waitTime time.Duration
 		backoff  time.Duration
 		attempts int
+		timer    = time.NewTimer(0)
 	)
 
 	// Start the notification runloop.
 	for {
 		// Increase the wait time for the next iteration.
 		backoff = waitTime + time.Duration(attempts)*time.Second
 		waitTime = 0
-		timer := time.NewTimer(backoff)
+
```

Comment:
I don't think we need the `waitTime = 0` above here.

Reply:
We want to reset the `waitTime` here which is really used in line 224 if we were able to connect the stream but upon first read it disconnected.